model/novinka.php

```diff
@@ -15,6 +15,10 @@ function autor() {
   function datum() {
     return date('j.n.', strtotime($this->r['vydat']));
   }
+  
+  function hlavniText () {
+    return preg_replace(self::$prvniObrazek, '', $this->text());
```

Comment:
Došlo mi, že toto asi vyhodí všechny obrázky v textu, ne jen první obrázek. Je potřeba proklikat víc blogů a ověřit, jestli se tak skutečně neděje. Pokud j, je potřeb zadat parametr limit, aby se to nahradilo jen jednou, viz dokumentace funkce http://cz1.php.net/manual/en/function.preg-replace.php